lib/recase/utils/enumerable.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Recase.Enumerable do
+  @moduledoc """
+  Helper module to convert enumerable keys recursively.
+  """
+
+  @doc """
```

Comment:
Let's mentions that it is not recursive. And does not work with nested structures.

Reply:
You write upper "We can also later add nested structures support!" it is already the case, the code support nested structures. so what do you mean by "Let's mentions that it is not recursive. "

Reply:
Oh, I see. This algorithm is actually recursive. I haven't seen it from the tests.
Sorry! 🙂